test/interval_tests/complex.jl

```diff
@@ -14,7 +14,7 @@ end
 
 @testset "Complex functions" begin
```

Comment:
(I just noticed I answered not in the correct place... Sorry.)

I agree that `a..b` is fast, that at most we loose an ulp, and that `@interval` is slow. My point is that the `±` notation is common in interval arithmetic and has a precise meaning, and thus we have to return the tightest interval possible. The `a..b` notation is sugar, and works fast and correctly to build examples where rounding is not important, like `0..1` or `0..∞`, or whenever getting the tightest interval is not crucial.